src/Avalonia.Controls/Repeater/ItemsRepeater.cs

```diff
@@ -681,8 +681,15 @@ private void OnLayoutChanged(AttachedLayout oldValue, AttachedLayout newValue)
             if (oldValue != null)
             {
                 oldValue.UninitializeForContext(LayoutContext);
-                oldValue.MeasureInvalidated -= InvalidateMeasureForLayout;
-                oldValue.ArrangeInvalidated -= InvalidateArrangeForLayout;
+
+                WeakEventHandlerManager.Unsubscribe<EventArgs, ItemsRepeater>(
+                    oldValue,
+                    nameof(newValue.MeasureInvalidated),
```

Comment:
This `newValue` is a bit confusing here. Can you use `nameof(AttachedLayout.MeasureInvalidated)` ?

Reply:
Sure. I used this because it was roughly analogous to `newValue.MeasureInvalidated += ...` but I don't really have any preference.